.gitignore

```diff
@@ -19,3 +19,9 @@ coverage.txt
 .idea/
 
 vendor/
+certs/
+cert-tool
+make-certificates.sh
```

Comment:
What is this and why we are ignoring some bash file?

Reply:
If this is for your custom development, keep it out of the dev repository and don't add it to gitignore please.